test/lit.cfg

```diff
@@ -135,7 +135,8 @@ if len(kleaver_extra_params) != 0:
 # Set absolute paths and extra cmdline args for KLEE's tools
 subs = [ ('%kleaver', 'kleaver', kleaver_extra_params),
   ('%klee','klee', klee_extra_params),
-  ('%ktest-tool', 'ktest-tool', '')
+  ('%ktest-tool', 'ktest-tool', ''),
+  ('%klee-replay', 'klee-replay', '')
```

Comment:
You need to introduce the `%klee-replay` substitution before `%klee`. Lit applies the substitutions in order so when it sees `%klee-replay` it will consider `%klee` before considering `%klee-replay`.

I don't know how the test wrote is even passing. So you should check why.